SwrveSDK/SwrveStorageManager.brs

```diff
@@ -75,59 +75,18 @@ function SwrveSaveStringToFile(str as String, filename as String) as Boolean
 	return success
 end function
 
-'Read string from tmp:/ storage' 
+'Read string from tmp:/ storage'
 function SwrveGetStringFromFile(filename as String) as String
 	return ReadAsciiFile(filename)
 end function
 
```

Comment:
The below functions where moved to the utils file as they are more common registry functions and are used by other files. This means you would need to import this script as well so simple operations.